packages/protocol/src/RebalancerManager.sol

```diff
@@ -54,6 +59,11 @@ contract RebalancerManager is IRebalancerManager, SystemAccessControl {
     if (!allowedExecutor[msg.sender]) {
       revert RebalancerManager__rebalanceVault_notValidExecutor();
     }
+
+    if (assets == 0) {
+      revert RebalancerManager__rebalanceVault_invalidAmount();
+    }
+
```

Comment:
Remove this condition. Check necessary changes to be able to rebalance just debt.

Reply:
![Screenshot 2023-05-05 at 12 28 16](https://user-images.githubusercontent.com/80268365/236446297-a5745b93-cca1-40f2-8baa-f110b972cb4a.png)
use this as an idea. ps: this is the rebalance function inside the borrowing vault.

All the _executeProviderAction should have this condition